src/components/docs/SwitchVersion.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { navigate } from 'gatsby';
+import { versions } from '../../../constants';
+
+const doChange = (e, location, currentVersion) => {
+  const targetVersion = !Number.isNaN(e.target.value) ? e.target.value : `v${e.target.value}`;
```

Comment:
Maybe we should have a function somewhere instead of having it duplicated everywhere in the code.